docs/site/Decorators.md

```diff
@@ -459,8 +459,8 @@ class MyComponent {
 }
 
 const ctx = new Context();
-ctx.bind('my-component').toClass(MyComponent);
-const component = ctx.getSync<MyComponent>('my-component');
+ctx.bind('my.component').toClass(MyComponent);
```

Comment:
should binding keys be using `.`'s? In a previous file you used `my-value`.

Reply:
Binding keys can definitely use `.`s (see `keys.ts` of the packages in the monorepo). But after having a look through them, they seem to be a bit over the place. For example, `openapi-v3` uses kebab-case with colons while others use camelCase.

We might need to decide what style we're going to advocate for naming binding keys